tasks/take-notes-on-the-command-line/src/main.rs

```diff
@@ -0,0 +1,44 @@
+extern crate time;
+
+use std::fs::{OpenOptions};
+use std::io::{BufReader, BufWriter};
+use std::io::prelude::*;
+use std::env;
+
+fn show_notes() {
+    let file = OpenOptions::new()
+        .read(true)
+        .write(true)
+        .create(true)
+        .open("NOTES.TXT")
+        .unwrap();
+    let mut buf_reader = BufReader::new(file);
+    let mut contents = String::new();
+    buf_reader.read_to_string(&mut contents).unwrap();
```

Comment:
consider returning an io::Result from the function instead of all these unwraps.